### DISCUSSION TITLE
Data inconsistency between script and reference

### DISCUSSION THREAD

**Discussion by reactingflow** [IDAES/idaes-pse#1376] [Q&A] [answered]
Hi， 

When i was learning and modifying the property data script named "egprod_ideal", i found that the expression and coefficients of molar heat capacity of water may be not consistent with those reported in the referrence, i.e. The Properties of Gases and Liquid, as shown below. As you can see, there are five parameters in the molar heat capacity expression in the referrence, but there are only four paramters in the script. 

![image](https://github.com/IDAES/idaes-pse/assets/151173251/2ef893c6-afd8-46b1-97bf-83609fe9f1f9)

![image](https://github.com/IDAES/idaes-pse/assets/151173251/63fe1875-5163-4a81-ae40-933e09f895fd)

![image](https://github.com/IDAES/idaes-pse/assets/151173251/f69e7762-4008-4fbd-9b9b-a26fdd846073)

**Comment by andrewlee94** [comment, marked as answer]
@reactingflow I believe the issue here is that the correlation used in The Properties of Gases and Liquids changed between the 4th and 5th editions. I suspect the values you are showing come from the 5th edition (or later), whilst this property package was written using the 4th edition (which only had 4 parameters).

The Modular Property Package libraries include support for the 3rd (RPP3), 4th (RPP4), and 5th (RPP5) editions, which is set on the `"enth_mol_ig_comp": RPP4` line, so if you prefer to use a more recent version you may do so by changing the value here.

**Reply by reactingflow** [reply]
Hi, Andrew @andrewlee94 

Thank you for your reminding me of the difference between editions. I checked the data of water in the forth edition of The Properties of Gases and Liquids. The data typed in the script may be those of H2S, as shown below.

![image](https://github.com/IDAES/idaes-pse/assets/151173251/9ca4cfbe-2bf8-499f-bd72-3672d10491a6)

**Reply by andrewlee94** [reply]
@reactingflow Thanks for looking into this, it does look like the parameter values were read from the wrong line. I've asked the maintainer of that example to look into this further.

**Reply by andrewlee94** [reply]
@reactingflow Would you have the time to take a look at this Pull Request: https://github.com/IDAES/examples/pull/95? The maintainer is working on fixing the incorrect values, and it would be good to get someone to double check the values to make sure we get them right this time.

**Reply by reactingflow** [reply]
Hi, Andrew @andrewlee94 

It is my pleasure to have the chance to contribute to the IDAES project. But I am new to IDAES, though I have some experience with Aspen.

**Reply by andrewlee94** [reply]
Thank you @reactingflow. First and foremost, if you could just double check the parameter values to make sure they are correct it would be a great help. Additionally, if you have any comments as a new user on what was valuable to you in the existing tutorials/examples, or more importantly things that were unclear or missing then this is a good opportunity to try and improve the content for future readers.

**Reply by reactingflow** [reply]
@andrewlee94 OK.
